lib/kubernetes-deploy/kubernetes_resource/custom_resource_definition.rb

```diff
@@ -2,8 +2,86 @@
 module KubernetesDeploy
   class CustomResourceDefinition < KubernetesResource
     TIMEOUT = 2.minutes
+    ROLLOUT_CONFIG_ANNOTATION = "kubernetes-deploy.shopify.io/monitor-instance-rollout"
     GLOBAL = true
 
+    class RolloutConfig
+      attr_reader :success_queries, :failure_queries
+
+      def initialize(rollout_config)
```

Comment:
If I'm reading this right the annotation value has to be a hash. Can we support `true` as well for just wanting the default behavior, seems more obvious than `{}`

Reply:
Fair point, I'll add that in